org.sofproject.alsa.topo/src/org/sofproject/alsa/topo/model/AlsaTopoGraph.java

```diff
@@ -536,7 +515,35 @@ public void serialize() throws CoreException, IOException {
 		}
 		outputFile.getParent().refreshLocal(1, null);
 	}
-
+	
+	@Override
+	public String getPipelineString() {
```

Comment:
It is a graph, that may consist of multiple pipelines, not a single one, so the method name should not refer to a pipeline.

Reply:
ok